nextflow.config

```diff
@@ -50,6 +50,10 @@ params {
     validationShowHiddenParams       = false
     validate_params                  = true
 
+
+    // Publish directories
+    assembly_directory_name = "assembly"
```

Comment:
Thank you for including this. I think it makes more sense to move setting the assembly and summary directory names into the `modules.config` file (where you set the `publishDir`).

Having these in `nextflow.config` means that they also make it into `nextflow_schema.json` and so would appear in the UI for setting parameters for the pipeline.. However, I don't think there's a particular reason to adjust these directory names at runtime, so they can remain as something that is set in `modules.config`.

Reply:
Makes sense. Updated.